content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+- driver := Person driving the asset physically, or entity controlling the asset programatically.
+- customer := Entity that uses the Ably Asset Tracking SDKs to build their product to solve the asset tracking problem for their users.
+- end user := Person that uses the product built by the customer and uses the asset tracking capabilities.
+
+Each asset being tracked must have a @trackableId@, and this must be globally unique within an Ably asset tracking account.
+
+The Ably Asset Tracking SDKs also allow for various battery optimization scenarios, so frequency of updates can be modified based on various parameters such as distance to destination, subscriber status, map zoom level, and existing battery levels.
+
+h2(#battery-optimization). Battery optimization scenarios
+
+One of the biggest challenges in asset tracking is combining high resolution of updates with low battery consumption on the asset tracking device. For example, in a delivery tracking use case you may want your users to have a smooth and realtime visualization of where their delivery is on the map, but at the same time sending realtime locations from the driver's device frequently will rapidly drain its battery.
+
+There are two key factors that impact asset tracking device battery usage:
+
+1. GPS sensor - obtaining location of the asset.
+2. Networking - sending location updates over network.
+
+It is hard to reduce the battery consumption of the GPS sensor in some situations, such as when the device is being used for navigation at the same time as tracking an asset. However, it is possible to adjust the networking aspect of resolution.
+
+The following scenarios are examples of how update resolution can be changed depending on the state of the asset and subscribers:
+
+* Updates can be sent to the subscriber more often when asset is closer to destination.
+* Updates don't need to happen as often when the user of the subscribing SDK is not observing the location of the asset, for example, the user of delivery app is not checking the app for the driver's location.
+* Different subscribers may have different preferences regarding the resolution of location updates.
+* When the driver's battery is running low, location updates can happen with lower frequency.
+
+Using the approaches similar to these, an optimal resolution of updates can be identified, which provides a good user experience when it is needed, such as when the driver is close to the destination, and the user is observing the location of the driver in the app. This approach also reduces battery usage when possible, for example when the driver is far from the destination and the user is not observing location updates.
+
+While developers can create their own code to implement battery optimization logic, Ably also provides a set of default constraints and policies, which are suitable for common use cases.
```

Comment:
Should we emphasise that Ably provides an extensible interface for building custom logic? Right now it sounds like the emphasis is on the default implementations, but I would argue that both are important: default implementation covers majority of basic use-cases, but SDKs are also very flexible and extensible because of the way we've built them, this extensibility is a feature.

Reply:
Yes, I've reworded.